src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -146,10 +147,16 @@ AndroidDeviceControllerWrapper * AndroidDeviceControllerWrapper::AllocateNew(
     setupParams.controllerVendorId             = static_cast<chip::VendorId>(controllerVendorId);
     setupParams.pairingDelegate                = wrapper.get();
     setupParams.operationalCredentialsDelegate = opCredsIssuer;
+    setupParams.defaultCommissioner            = &wrapper->mAutoCommissioner;
     initParams.fabricIndependentStorage        = wrapperStorage;
 
     wrapper->mGroupDataProvider.SetStorageDelegate(wrapperStorage);
 
+    CommissioningParameters params = wrapper->mAutoCommissioner.GetCommissioningParameters();
+    params.SetFailsafeTimerSeconds(failsafeTimerSeconds);
+    params.SetAttemptWiFiNetworkScan(attemptNetworkScanWiFi);
+    params.SetAttemptThreadNetworkScan(attemptNetworkScanThread);
```

Comment:
I'm not sure I understand this code.  We're setting up some state on `params`, but nothing uses `params` as far as can tell.

Reply:
fixed to use a single method (AutoCommissioner.SetCommissioningParameters) to perform all updates to the commissioning parameters it uses.